queries.sql

```diff
@@ -1,3 +1,34 @@
 /*Queries that provide answers to the questions from all projects.*/
-
 SELECT * from animals WHERE name = 'Luna';
+-- Find all animals whose name ends in "mon".
+ SELECT *
+ FROM animals
+ WHERE name LIKE '%mon';
+-- List the name of all animals born between 2016 and 2019.
+SELECT name 
+FROM animals
+WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+-- List the name of all animals that are neutered and have less than 3 escape attempts.
+SELECT name 
+FROM animals
+WHERE neutered = 'true' AND escape_attempts < 3;
+-- List date of birth of all animals named either "Agumon" or "Pikachu".
+SELECT * 
+FROM date_of_birth
+WHERE name = 'Agumon' or name ='Pikachu';
```

Comment:
Please add a query here to select the date of birth of all animals named either "Agumon" or "Pikachu", so you need to select `date_of_birth` from the `animals` table here.